index.html

```diff
@@ -66,7 +66,258 @@ <h1 class="header-title ">I'm Abigail T Glad to see you!
+                  
+                </div>
+              </div>
+            </li>
+          </ul>
+        </div>
+    </section>
+
+    <!-- my third design -->
+
+    <section class="third-design">
+      <div class="design-outer">
+        <ul class="design-inner">
+          <li>
+            <div class="first-card">
+              <img
+                class="first-work-one"
+                src="images/third-work.png"
+                alt="First Work"
+              />
+              <div class="card-content">
+                <h3 class="card-title">Tonic</h3>
+                <div class="card-info">
+                  <p class="card-info-item-canopy">CANOPY</p>
+                  <img class="circle-img" src="./images/circle.png" alt="Circle">
+                  <p class="card-info-item">Back End Dev</p>
+                  <img class="circle-img" src="./images/circle.png" alt="Circle">
+                  <p class="card-info-item">2015</p>
+                </div>
+
+                <p class="card-description">
+                  A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <ul class="card-list">
+                  <li class="card-list-tags-html">html</li>
+                  <li class="card-list-tags-css">css</li>
+                  <li class="card-list-tags-js">Javascript</li>
+                </ul>
+
+                <a class="card-link" href="#">See Project</a>
+                
+              </div>
+            </div>
+          </li>
+        </ul>
+      </div>
+  </section>
+
+  <!-- fourth design -->
+
+  <section class="fourth-design">
+    <div class="design-outer">
+      <ul class="design-inner">
+        <li>
+          <div class="first-card">
+            <img
+              class="first-work-one"
+              src="images/fourth-work.png"
+              alt="First Work"
+            />
+            <div class="card-content">
+              <h3 class="card-title">Multi-Post Stories</h3>
+              <div class="card-info">
+                <p class="card-info-item-canopy">CANOPY</p>
+                <img class="circle-img" src="./images/circle.png" alt="Circle">
+                <p class="card-info-item">Back End Dev</p>
+                <img class="circle-img" src="./images/circle.png" alt="Circle">
+                <p class="card-info-item">2015</p>
+              </div>
+
+              <p class="card-description">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+              </p>
+
+              <ul class="card-list">
+                <li class="card-list-tags-html">html</li>
+                <li class="card-list-tags-css">css</li>
+                <li class="card-list-tags-js">Javascript</li>
+              </ul>
+
+              <a class="card-link" href="#">See Project</a>
+              
+            </div>
+          </div>
+        </li>
+      </ul>
+    </div>
+</section>
```

Comment:
- [ ] Please note that all your cards should be in one section. Your cards can be articles in a section or just divs in a section. Also, you are to align the contents of the section (that is the cards) using CSS grid as demanded by project requirements. Kindly fix this, please.

Reply:
resolved issue 
thank you